.github/workflows/cloudbeat-ci.yaml

```diff
@@ -181,17 +181,41 @@ jobs:
 
       - name: Comment test success result
         uses: marocchino/sticky-pull-request-comment@v2
-        if: always()
+        if: steps.run_tests.outcome == 'success'
         with:
           header: Test Results
-          recreate: true
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           message: |
             ## Cloudbeat CI :robot:
-            Integration tests status: ${{ steps.run_tests.outcome }}
+            Integration tests passed! :green_heart:
             Tests Summary:
             ```python
             ${{env.summary}}
             ```
-            Link to detailed report: https://${{ github.repository_owner }}.github.io/cloudbeat/${{ github.run_number }}
+            Link to detailed report: https://${{ github.repository_owner }}.github.io/cloudbeat/${{ github.run_number }}
+
+      - name: Comment test skipped result
```

Comment:
The name the failed & skipped comment steps are mixed up.

Reply:
by design, I will change the name though

Reply:
oh I meant something else nvm